go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/microsoft/retina
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.2
```

Comment:
actually can you revert this?
I don't really understand when the toolchain directive gets added, but I think this should remain `go 1.22` so we use the one in the builder image. If we have to have toolchain for some reason, it needs to be at least `1.22.3` for CVE fixes.